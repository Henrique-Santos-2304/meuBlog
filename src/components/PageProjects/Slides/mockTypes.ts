export const mockTypes = {
  cardsSlides: [
    {
      miniDescription: "Uma Descrição dos Slides",
      title: "Ver projetos",
      __typename: "nada",
      buttons: [
        { url: "ir.com", label: "fui" },
        { url: "ir.com2", label: "fui2" },
      ],
      imageFront: {
        alt: "descrição da imagem",
        ariaLabel: "essa imagem",
        imageUser: {
          url: "img/img.png",
        },
      },
    },
  ],
};
