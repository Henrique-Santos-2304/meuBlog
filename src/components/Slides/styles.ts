import styled from "styled-components";

export const Icons = styled.div`
  &.slick-prev {
    left: -45px;
  }
  &::before {
    font-size: 4rem;
  }
`;
export const Dots = styled.div`
  bottom: -4rem;
  ul {
    &:last-child {
      color: orange;
    }
    li {
      color: orange;
      &.slick-active {
        button {
          &::before {
            color: orange;
          }
        }
      }
      button {
        &::before {
          font-size: 2rem;
        }
      }
    }
  }
`;
