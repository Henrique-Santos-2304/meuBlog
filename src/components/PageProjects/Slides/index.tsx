/* eslint-disable @typescript-eslint/no-explicit-any */
import CardBox from "components/PageProjects/CardBox";
import { cardSlidesProps } from "graphql/typesFinal/queryProjects";
import React from "react";
import Slider from "react-slick";
import * as S from "./styles";

type props = {
  cardsSlides: cardSlidesProps[];
};

const Slides = ({ cardsSlides }: props) => {
  const settings = {
    dots: true,
    centerMode: true,
    infinite: true,
    autoplay: true,
    autoplaySpeed: 3000,
    cssEase: "linear",
    speed: 1000,
    slidesToShow: 1,
    slidesToScroll: 1,
    initialSlide: 2,
    arrows: false,
    responsive: [
      {
        breakpoint: 620,
        settings: {
          slidesToShow: 1,
          slidesToScroll: 1,
          initialSlide: 2,
        },
      },
      {
        breakpoint: 550,
        settings: {
          slidesToShow: 1.03,
          centerMode: false,
          slidesToScroll: 1,
          initialSlide: 2,
        },
      },
    ],
    appendDots: (dots: any) => (
      <S.Dots>
        <ul aria-label="BotÃµes para trocar as fotos"> {dots} </ul>
      </S.Dots>
    ),
  };

  return (
    <Slider {...settings}>
      {cardsSlides.map((cards, index) => (
        <CardBox
          key={index}
          url={cards.imageFront.imageUser.url}
          title={cards.title}
          subDescription={cards.miniDescription}
          alt={cards.imageFront.alt}
          buttons={cards.buttons}
        />
      ))}
    </Slider>
  );
};
export default Slides;
