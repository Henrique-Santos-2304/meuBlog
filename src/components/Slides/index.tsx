/* eslint-disable @typescript-eslint/no-explicit-any */
import CardBox from "components/CardBox";
import React from "react";
import Slider from "react-slick";
import { images } from "../../pages/projects/images/images";
import * as S from "./styles";

type imageProps = {
  url: string;
  title: string;
  subDescription: string;
};
function SamplePrevArrow(props: {
  className?: string;
  style?: any;
  onClick?: () => void;
}) {
  const { className, style, onClick } = props;
  return (
    <S.Icons
      className={className}
      style={{ ...style }}
      onClick={onClick}
      id="left"
    />
  );
}
function SampleNextArrow(props: {
  className?: string;
  style?: any;
  onClick?: () => void;
}) {
  const { className, style, onClick } = props;
  return (
    <S.Icons className={className} style={{ ...style }} onClick={onClick} />
  );
}
const Slides = () => {
  const settings = {
    dots: true,
    infinite: false,
    speed: 800,
    slidesToShow: 2.4,
    slidesToScroll: 2,
    responsive: [
      {
        breakpoint: 1100,
        settings: {
          slidesToShow: 2.2,
          slidesToScroll: 2,
          initialSlide: 2,
          arrows: false,
        },
      },
      {
        breakpoint: 768,
        settings: {
          slidesToShow: 2.2,
          slidesToScroll: 2,
          initialSlide: 2,
        },
      },
      {
        breakpoint: 480,
        settings: {
          slidesToShow: 1.1,
          slidesToScroll: 1,
          arrows: false,
        },
      },
    ],
    appendDots: (dots: any) => (
      <S.Dots>
        <ul> {dots} </ul>
      </S.Dots>
    ),
    nextArrow: <SampleNextArrow />,
    prevArrow: <SamplePrevArrow />,
  };

  return (
    <Slider {...settings}>
      {images.map((image: imageProps, index) => (
        <CardBox
          key={index}
          url={image.url}
          title={image.title}
          subDescription={image.subDescription}
        />
      ))}
    </Slider>
  );
};
export default Slides;
