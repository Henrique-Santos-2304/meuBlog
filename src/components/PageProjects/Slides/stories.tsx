import { Story, Meta } from "@storybook/react";
import Slides from ".";

export default {
  title: "Slides",
  component: Slides,
} as Meta;

export const Basic: Story = (args) => <Slides {...args} />;

Basic.args = {
  title: "title basic",
  description: "description basic",
};

export const Default: Story = (args) => <Slides {...args} />;
