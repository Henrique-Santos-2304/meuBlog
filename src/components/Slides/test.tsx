import "match-media-mock";
import Slides from ".";
import { renderTheme } from "utils/testRenderTheme";

describe("<Slides />", () => {
  it("should render ", () => {
    const { container } = renderTheme(<Slides />);
    expect(container).toMatchSnapshot();
  });
});
