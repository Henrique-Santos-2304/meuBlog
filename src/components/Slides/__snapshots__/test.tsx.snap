// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Slides /> should render  1`] = `
.c2 {
  color: #E5E7EB;
  font-size: 2.0rem;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  color: #06092b;
  font-size: 1.4rem;
  position: absolute;
  bottom: 2rem;
  left: 50%;
  -webkit-transform: translateX(-50%);
  -ms-transform: translateX(-50%);
  transform: translateX(-50%);
  overflow: hidden;
}

.c0.slick-prev {
  left: -45px;
}

.c0::before {
  font-size: 4rem;
}

.c3 {
  bottom: -4rem;
}

.c3 ul:last-child {
  color: orange;
}

.c3 ul li {
  color: orange;
}

.c3 ul li.slick-active button::before {
  color: orange;
}

.c3 ul li button::before {
  font-size: 2rem;
}

<div>
   
  <div
    class="slick-slider slick-initialized"
    dir="ltr"
  >
    <div
      class="c0 slick-arrow slick-prev slick-disabled"
      id="left"
      style="display: block;"
    />
    <div
      class="slick-list"
    >
      <div
        class="slick-track"
        style="opacity: 1; transform: translate3d(0px, 0px, 0px);"
      >
        <div
          aria-hidden="false"
          class="slick-slide slick-active slick-current"
          data-index="0"
          style="outline: none; width: 0px;"
          tabindex="-1"
        >
          <div>
            <div
              class="css-15mj58l"
            >
              <img
                class="chakra-image css-14lr2s0"
                src="https://images.unsplash.com/photo-1496449903678-68ddcb189a24?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1470&q=80"
              />
              <h2
                class="chakra-heading css-vqu1z5"
              >
                Projeto Bar
              </h2>
              <p
                class="chakra-text css-23t00v"
              >
                Um projeto sobre um Bar, com integração e compras online
              </p>
              <div
                class="c1"
              >
                <div
                  class="chakra-button__group css-gg6mva"
                  role="group"
                >
                  <button
                    class="chakra-button css-10tw6y9"
                    type="button"
                  >
                    Ver projeto
                  </button>
                  <button
                    class="chakra-button css-1hw5e2v"
                    type="button"
                  >
                    <span
                      class="chakra-button__icon css-1wh2kri"
                    >
                      <div
                        aria-hidden="true"
                        class="c2"
                        focusable="false"
                      >
                        <svg
                          fill="currentColor"
                          height="1em"
                          stroke="currentColor"
                          stroke-width="0"
                          viewBox="0 0 496 512"
                          width="1em"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                          />
                        </svg>
                      </div>
                    </span>
                    Ver Código
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden="false"
          class="slick-slide slick-active"
          data-index="1"
          style="outline: none; width: 0px;"
          tabindex="-1"
        >
          <div>
            <div
              class="css-15mj58l"
            >
              <img
                class="chakra-image css-14lr2s0"
                src="https://images.unsplash.com/photo-1499854413229-6d1c92ff39ef?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=931&q=80"
              />
              <h2
                class="chakra-heading css-vqu1z5"
              >
                Sobre Sol
              </h2>
              <p
                class="chakra-text css-23t00v"
              >
                Um projeto sobre o sol, com noticias e informações sobre o sol
              </p>
              <div
                class="c1"
              >
                <div
                  class="chakra-button__group css-gg6mva"
                  role="group"
                >
                  <button
                    class="chakra-button css-10tw6y9"
                    type="button"
                  >
                    Ver projeto
                  </button>
                  <button
                    class="chakra-button css-1hw5e2v"
                    type="button"
                  >
                    <span
                      class="chakra-button__icon css-1wh2kri"
                    >
                      <div
                        aria-hidden="true"
                        class="c2"
                        focusable="false"
                      >
                        <svg
                          fill="currentColor"
                          height="1em"
                          stroke="currentColor"
                          stroke-width="0"
                          viewBox="0 0 496 512"
                          width="1em"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                          />
                        </svg>
                      </div>
                    </span>
                    Ver Código
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden="false"
          class="slick-slide slick-active"
          data-index="2"
          style="outline: none; width: 0px;"
          tabindex="-1"
        >
          <div>
            <div
              class="css-15mj58l"
            >
              <img
                class="chakra-image css-14lr2s0"
                src="https://images.unsplash.com/photo-1508138221679-760a23a2285b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=774&q=80"
              />
              <h2
                class="chakra-heading css-vqu1z5"
              >
                Projeto Aviões
              </h2>
              <p
                class="chakra-text css-23t00v"
              >
                Um Projetos sobre os aviões na floresta
              </p>
              <div
                class="c1"
              >
                <div
                  class="chakra-button__group css-gg6mva"
                  role="group"
                >
                  <button
                    class="chakra-button css-10tw6y9"
                    type="button"
                  >
                    Ver projeto
                  </button>
                  <button
                    class="chakra-button css-1hw5e2v"
                    type="button"
                  >
                    <span
                      class="chakra-button__icon css-1wh2kri"
                    >
                      <div
                        aria-hidden="true"
                        class="c2"
                        focusable="false"
                      >
                        <svg
                          fill="currentColor"
                          height="1em"
                          stroke="currentColor"
                          stroke-width="0"
                          viewBox="0 0 496 512"
                          width="1em"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                          />
                        </svg>
                      </div>
                    </span>
                    Ver Código
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden="true"
          class="slick-slide"
          data-index="3"
          style="outline: none; width: 0px;"
          tabindex="-1"
        >
          <div>
            <div
              class="css-15mj58l"
            >
              <img
                class="chakra-image css-14lr2s0"
                src="https://images.unsplash.com/photo-1508138221679-760a23a2285b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=774&q=80"
              />
              <h2
                class="chakra-heading css-vqu1z5"
              >
                Projeto Aviões
              </h2>
              <p
                class="chakra-text css-23t00v"
              >
                Um Projetos sobre os aviões na floresta
              </p>
              <div
                class="c1"
              >
                <div
                  class="chakra-button__group css-gg6mva"
                  role="group"
                >
                  <button
                    class="chakra-button css-10tw6y9"
                    type="button"
                  >
                    Ver projeto
                  </button>
                  <button
                    class="chakra-button css-1hw5e2v"
                    type="button"
                  >
                    <span
                      class="chakra-button__icon css-1wh2kri"
                    >
                      <div
                        aria-hidden="true"
                        class="c2"
                        focusable="false"
                      >
                        <svg
                          fill="currentColor"
                          height="1em"
                          stroke="currentColor"
                          stroke-width="0"
                          viewBox="0 0 496 512"
                          width="1em"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                          />
                        </svg>
                      </div>
                    </span>
                    Ver Código
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden="true"
          class="slick-slide"
          data-index="4"
          style="outline: none; width: 0px;"
          tabindex="-1"
        >
          <div>
            <div
              class="css-15mj58l"
            >
              <img
                class="chakra-image css-14lr2s0"
                src="https://images.unsplash.com/photo-1499854413229-6d1c92ff39ef?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=931&q=80"
              />
              <h2
                class="chakra-heading css-vqu1z5"
              >
                Sobre Sol
              </h2>
              <p
                class="chakra-text css-23t00v"
              >
                O pedaço padrão de Lorem Ipsum usado desde 1500 é reproduzido abaixo para os interessados. As seções 1.10.32 e 1.10.33 de de Finibus Bonorum et Malorum por Cícero também são reproduzidas em sua forma original exata, acompanhadas por versões em inglês da tradução de 1914 por H. Rackham.
              </p>
              <div
                class="c1"
              >
                <div
                  class="chakra-button__group css-gg6mva"
                  role="group"
                >
                  <button
                    class="chakra-button css-10tw6y9"
                    type="button"
                  >
                    Ver projeto
                  </button>
                  <button
                    class="chakra-button css-1hw5e2v"
                    type="button"
                  >
                    <span
                      class="chakra-button__icon css-1wh2kri"
                    >
                      <div
                        aria-hidden="true"
                        class="c2"
                        focusable="false"
                      >
                        <svg
                          fill="currentColor"
                          height="1em"
                          stroke="currentColor"
                          stroke-width="0"
                          viewBox="0 0 496 512"
                          width="1em"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                          />
                        </svg>
                      </div>
                    </span>
                    Ver Código
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden="true"
          class="slick-slide"
          data-index="5"
          style="outline: none; width: 0px;"
          tabindex="-1"
        >
          <div>
            <div
              class="css-15mj58l"
            >
              <img
                class="chakra-image css-14lr2s0"
                src="https://images.unsplash.com/photo-1496449903678-68ddcb189a24?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1470&q=80"
              />
              <h2
                class="chakra-heading css-vqu1z5"
              >
                Projeto Bar
              </h2>
              <p
                class="chakra-text css-23t00v"
              >
                Um projeto sobre um Bar, com integração e compras online
              </p>
              <div
                class="c1"
              >
                <div
                  class="chakra-button__group css-gg6mva"
                  role="group"
                >
                  <button
                    class="chakra-button css-10tw6y9"
                    type="button"
                  >
                    Ver projeto
                  </button>
                  <button
                    class="chakra-button css-1hw5e2v"
                    type="button"
                  >
                    <span
                      class="chakra-button__icon css-1wh2kri"
                    >
                      <div
                        aria-hidden="true"
                        class="c2"
                        focusable="false"
                      >
                        <svg
                          fill="currentColor"
                          height="1em"
                          stroke="currentColor"
                          stroke-width="0"
                          viewBox="0 0 496 512"
                          width="1em"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                          />
                        </svg>
                      </div>
                    </span>
                    Ver Código
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c0 slick-arrow slick-next"
      style="display: block;"
    />
    <div
      class="c3 slick-dots"
    >
      <ul>
         
        <li
          class="slick-active"
        >
          <button>
            1
          </button>
        </li>
        <li
          class=""
        >
          <button>
            2
          </button>
        </li>
        <li
          class=""
        >
          <button>
            3
          </button>
        </li>
         
      </ul>
    </div>
  </div>
</div>
`;
